XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1205,6 +1205,26 @@
             "RelativePanel"
           ]
         },
+        {
+          "UniqueId": "ItemsRepeater",
+          "Title": "ItemsRepeater",
+          "Subtitle": "A flexible, primitive control for data-driven layouts.",
+          "ImagePath": "ms-appx:///Assets/ListView.png",
+          "Description": "The ItemsRepeater is like a markup-based loop that supports virtualization.",
+          "Content": "<p>A <b>ItemsRepeater</b> is more basic than an <b>ItemsControl</b>.  It does not have default styling or a control template.  It can contain a collection of items of any type. To populate the view, set the <b>ItemsSource</b> property to a data source.</p><p>Set an <b>ItemTemplate</b> to define the look of individual items.</p><p>And, set the <b>Layout</b> to define how items are sized and positioned.</p><p>Look at the <i>ItemsRepeaterPage.xaml</i> file in Visual Studio to see the full code for this page.</p>",
+          "IsNew": true,
```

Comment:
Since this is a preview feature (for now), please mark this as "IsPreview": true, not "IsNew": true.